src/platforms/common/configuration/releases.mdx

```diff
@@ -25,8 +25,8 @@ Include a release ID (often called a "version") when you configure your client S
 
 The release name cannot:
 
-- contain newlines or spaces
-- use a forward slash (`/`), back slash (`\`), period (`.`), or double period (`..`)
+- contain newlines, tabulator characters, forward slashes(`/`) or back slashes(`\`) 
```

Comment:
```suggestion
- contain newlines, tabulator characters, forward slashes(`/`), or back slashes(`\`) 
```

Reply:
Thank you! This fixes a few other open issues as well.